e2e/feature/dashboard/header/step_definitions/headerFeature_steps.ts

```diff
@@ -0,0 +1,99 @@
+import { browser, element, by } from 'protractor';
+import { HeaderPage } from '../../../../page-objects/header.po';
+import { Capdash2Page } from '../../../../page-objects/app.po';
+import { E2EHelpers } from '../../../../support/e2eHelpers';
+
+import { defineSupportCode } from 'cucumber';
+
+import { expect } from 'chai';
+
+defineSupportCode(({Given, When, Then}) => {
+  const header: HeaderPage = new HeaderPage();
+  const app: Capdash2Page = new Capdash2Page();
+
+  const menus = [
```

Comment:
Keeping this array in sync with the Examples section of the Scenario Outline is additional maintenance and should not be necessary.